tools/generator-go-sdk/generator/helpers_test.go

```diff
@@ -1,6 +1,7 @@
 package generator
```

Comment:
fwiw the `sdk` package has a `testhelpers` package with a canonical reference for this now